turingschool/savile
# WIP: Add default color for paragraph as util

#### What does this PR do?

This PR add a `util` of a default class for the body tag - it has one color declaration for paragraphs. Note that this PR does not include anything for the documentation site; I'm not sure how that should ultimately look, but was focusing on "can we make a default class and get it in an app? What road bumps will arise?"

#### Where should the reviewer start?

Not a lot of code to look at; you probably want to look at it in action - [the WIP PR for implementation on turing_io is here](https://github.com/turingschool/turing_io/pull/317). 

Question for you: Does putting that in as a util sound right? I followed the same pattern for docs generator and ran a build; that was probably not the ideal situation, but the focus here was to get it working - next step will be deciding how that should look in docs. Just want you to know I don't love this at the moment!! 

#### How should this be manually tested?

- Run Savile locally - this branch
- Run turing_io locally - the branch associated with its PR linked above
- Turing_io will hit local Savile, and use the default class. You could compare live Turing_io with local and see the differences in gray paragraphs!

#### What are the relevant tickets?
https://3.basecamp.com/3494409/buckets/19192671/todolists/3458194994

#### Screenshots (if appropriate)
Below are some screenshots of the BEFORE:
![image](https://user-images.githubusercontent.com/25447342/107854143-389ad700-6dd7-11eb-92aa-28b33d6c1328.png)
![image](https://user-images.githubusercontent.com/25447342/107854155-49e3e380-6dd7-11eb-90c7-71cb61230538.png)
![image](https://user-images.githubusercontent.com/25447342/107854175-6d0e9300-6dd7-11eb-87bb-d5500a654e2c.png)

Below are some screenshots of the AFTER:
![image](https://user-images.githubusercontent.com/25447342/107854147-405a7b80-6dd7-11eb-9365-408511ba0bb3.png)
![image](https://user-images.githubusercontent.com/25447342/107854158-4e100100-6dd7-11eb-9727-4f75174f59d8.png)
![image](https://user-images.githubusercontent.com/25447342/107854179-7435a100-6dd7-11eb-9ec6-96937f826469.png)
